plugins/permission-react/src/apis/IdentityPermissionApi.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DiscoveryApi, IdentityApi } from '@backstage/core-plugin-api';
+import { PermissionApi } from './PermissionApi';
+import {
+  AuthorizeRequest,
+  AuthorizeResponse,
+  PermissionClient,
+} from '@backstage/plugin-permission-common';
+import { Config } from '@backstage/config';
+
+/**
+ * The default implementation of the PermissionApi, which simply calls the authorize method of the given
+ * {@link @backstage/plugin-permission-common#PermissionClient}.
+ * @public
+ */
+export class IdentityPermissionApi implements PermissionApi {
+  constructor(
```

Comment:
I'd say it's best to either make this private, or to also have it use an options object for passing in the deps. An long arg array is a PITA x)

Reply:
Made it private - and because it is private, I think it's fine to leave the arguments as-is.